### PR TITLE
Eol comment

### LINKED ISSUE #55: standard-format doesn't remove trailing semicolons if they're followed by a comment
```sh
$ standard-format --version
1.3.6
```

### Steps to reproduce:
1. Create a new file, _foo.js_, w/ the following code:

    ```js
    var config = {
      consumer_key: consumer_key,
      access_token: access_token
    }; // useful comment
```

2. Run the following Terminal command: `$ standard-format -w foo.js`, or the following code to format and lint the code using **standard**:
    ```sh
    $ standard --format foo.js
    standard: Use JavaScript Standard Style (https://github.com/feross/standard)
      /Users/pdehaan/dev/tmp/pocketapi-test/foo.js:12:2: Extra semicolon.
    $ standard --version # 3.7.0
```

### Actual results:
The semicolon isn't removed during auto-formatting.

### Expected results:
The semicolon is removed.

### PR DESCRIPTION
**Pull request by Flet**
fixes #55 

I did not have a chance to add more tests, but:
image

Will this work @bcomnes?

### CODE REVIEW

**Comment by bcomnes** [comment]
I think for now, yes.  Would be cool to eventually minimize regexs though.

File: test/singleline.js [hunk 1]
```diff
@@ -92,14 +92,14 @@ var transforms = [
...
-  },
-  {
     str: 'if(true){}\n',
     expect: 'if (true) {}\n',
     msg: 'Space after if'
+  },
+  {
+    str: 'var x = 123; // Useful comment\n',
+    expect: 'var x = 123 // Useful comment\n',
+    msg: 'Remove uneeded trailing semicolons that are followed by a comment'
```
Comment (pdehaan) [inline test/singleline.js]: FWIW, it should be "unneeded".

**Comment by Flet** [comment]
@bcomnes agreed, regex should not be the final answer here... I think a new plugin for esformatter may be required to solve this without regex as I don't think `esformatter-semicolons` really does it.

Hopefully efforts to accomplish #26 will replace this soon :)

File: test/singleline.js (hunk 1 above)
Comment (bcomnes) [inline test/singleline.js]: Whoops!  Should probably fix that

Comment (Flet) [inline test/singleline.js]: luuks fyn two mee.

Comment (pdehaan) [inline test/singleline.js]: Considering the filename is "sigleline.js", maybe fixing the typo is unecessary.

Comment (bcomnes) [inline test/singleline.js]: This is what I get for working on a high res monitor. I'm so sorry everyone.

Comment (pdehaan) [inline test/singleline.js]: Haha, no worries. I submitted #63 which covers the typo and renames the test file.

**Pull request merged**